### ISSUE TITLE
Chapter 10 Error ( Accuracy is always 0% i.e constant)

### ISSUE THREAD

**Issue by mahdirayeni** [PacktPublishing/PyTorch-Computer-Vision-Cookbook#8]
I have trained your model on the hmdb51 dataset but the accuracy always remains contant even for different epochs.

![Screenshot (254)](https://user-images.githubusercontent.com/55917493/98228753-433f0800-1f6e-11eb-9593-a41ca5bbc271.png)

Please guide me further.

**Comment by mave5** [comment]
Seems like the flag "sanity_check" is set to True. You need to set the flag to False and try again.
